packages/crayons-core/src/components/timepicker/timepicker.tsx

```diff
@@ -26,22 +34,19 @@ export class Timepicker {
    * State for all the time values
    */
   @State() timeValues: any[] = [];
+  @State() langModule: any;
 
   /**
    * Format in which time values are populated in the list box. If the value is hh:mm p, the time values are in the 12-hour format. If the value is hh:mm, the time values are in the 24-hr format.
+   * The default value will be set based on the locale time format.
    */
-  @Prop() format: 'hh:mm a' | 'HH:mm' = 'hh:mm a';
+  @Prop({ mutable: true }) format: string;
 
   /**
    * Set true to disable the element
    */
   @Prop() disabled = false;
 
-  /**
-   * Represent the intervals and can be a number or array of numbers representing the minute values in an hour
-   */
-  @State() isMeridianFormat?: boolean = this.format === 'hh:mm a';
-
   /**
    * Time output value
```

Comment:
Are we expecting that value should always be of `HH:mm` format only ?

Reply:
No the value is expected to of same format as that of the `format` or `locale` attributes. If the `value` attribute and one of the option's value in the select matches it will select that value otherwise it wont.